acceptance-tests/helpers/service_broker.go

```diff
@@ -81,15 +89,14 @@ func BrokerFromDir(dir string) Option {
 
 func DefaultBroker() ServiceBroker {
 	return ServiceBroker{
-		Name: "broker-cf-test",
+		Name: "csb-gblue",
```

Comment:
Think this slipped in :) should be `broker-cf-test`

Reply:
Good catch